script/start-odc.sh

```diff
@@ -98,6 +98,11 @@ function init_parameters() {
     fi
     export LOCAL_HOSTNAME=$(hostname -I | awk -F ' ' '{print $1}')
     export ODC_PROFILE_MODE="${profile}"
+
+    # set heap MaxRAMPercentage=60.0 if ODC_TASK_RUN_MODE is K8S
+    if [[ "${ODC_TASK_RUN_MODE}" == "K8S" ]]; then
```

Comment:
Should add `"${ODC_BOOT_MODE}" == "TASK_EXECUTOR"` ?

Reply:
yes, MaxRAMPercentage is 45% in process mode, job running with odc-server,  in k8s mode, task be running in pod, so MaxRAMPercentage should keep old value